flask/cli.py

```diff
@@ -341,43 +341,40 @@ def _load_plugin_commands(self):
 
     def get_command(self, ctx, name):
         self._load_plugin_commands()
-
-        # We load built-in commands first as these should always be the
```

Comment:
Is there discussion about this statement? It appears Armin already had that usecase in mind and intentionally wants to discourage overriding builtin commands.